tests/pages/contact.py

```diff
@@ -71,6 +71,15 @@ def spaces(self):
         return [self.Space(self, root=el) for el in
             self.find_elements(self._spaces_locator)]
 
+    @property
+    def _is_map_ready(self):
+        return self.displayed_map_pins == len(self.spaces)
+
+    def wait_for_page_to_load(self):
+        super(ContactPage, self).wait_for_page_to_load()
+        self.wait.until(lambda s: self._is_map_ready)
```

Comment:
The `_is_map_ready` is only used here as far as I can tell. Is there a reason not to just put the condition directly in here?